draft-ietf-tls-ctls.md

```diff
@@ -333,6 +338,9 @@ treatment, as opposed to hex values.
 Static vectors (see {{static-vectors}}):
 
 * `Extension.extension_data` for any extension in `expected_extensions` whose value is self-delimiting (e.g., fixed length).  This applies only to the corresponding message.
+* `Extension.extension_data` for any additional extension whose type is in
+  `known_self_delimiting_extensions`, or is listed in
+  {{Section 4.2 of !RFC8446}} except `padding`.
```

Comment:
If you want to avoid requiring prior agreement on which extensions are self-delimiting, this must also apply to `expected_extensions`.

```suggestion
* `Extension.extension_data` for any extension whose type is in
  `omit_length`, or is listed in
  {{Section 4.2 of !RFC8446}} except `padding`.  This applies only to the corresponding message.
```